### PR TITLE
fix: Fix import hard wallet.

### PR DESCRIPTION
**Pull request by yanguoyu**

### CODE REVIEW

File: packages/neuron-wallet/src/controllers/wallets.ts [hunk 1]
```diff
@@ -34,6 +34,20 @@ import AddressParser from '../models/address-parser'
 import MultisigConfigModel from '../models/multisig-config'
 import { generateRPC } from '../utils/ckb-rpc'
```
Comment (twhy) [inline packages/neuron-wallet/src/controllers/wallets.ts:36]: We need to add some comments explains why the `compressPublicKey()` is needed when importing hardware wallets.

File: packages/neuron-wallet/src/controllers/wallets.ts [hunk 2]
```diff
@@ -34,6 +34,20 @@ import AddressParser from '../models/address-parser'
...
 
+const UNCOMPRESSED_KEY_LENGTH = 130
```
Comment (twhy) [inline packages/neuron-wallet/src/controllers/wallets.ts:37]: We need to add some comments explains why the `compressPublicKey()` is needed when importing hardware wallets.

**Review by twhy — APPROVED** [review]

**Comment by twhy** [comment]
@yanguoyu  Thank you for fixing this!

**Comment by Keith-CY** [comment]
Please add tests for this case

**Comment by silySuper** [comment]
s plus import wallet name the wallet is fixed now.
But when find device,it throw an error ,does this error related to this issue?
截屏2024-05-28 15 02 45

File: packages/neuron-wallet/src/controllers/wallets.ts (hunk 2 above)
Comment (yanguoyu) [reply packages/neuron-wallet/src/controllers/wallets.ts]: Added

**Comment by yanguoyu** [comment]
> Please add tests for this case



> Please add tests for this case

Added

**Review by silySuper — APPROVED** [review]

File: packages/neuron-wallet/tests/services/hardware/ledger.test.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    })
+
+    it('if the return pk is zip', async () => {
+      getWalletExtendedPublicKeyMock.mockResolvedValueOnce({ public_key: zipPk, chain_code: '0x' })
+      const ledger = new Ledger({} as any)
+      await ledger.connect()
+      const res = await ledger.getExtendedPublicKey()
+      expect(res.publicKey).toBe(zipPk)
+    })
+  })
```
Comment (Keith-CY) [inline packages/neuron-wallet/tests/services/hardware/ledger.test.ts:38]:
```suggestion
  describe('test getExtendedPublicKey', () => {
    it('if the return pk is not compressed', async () => {
      getWalletExtendedPublicKeyMock.mockResolvedValueOnce({ public_key: pk, chain_code: '0x' })
      const ledger = new Ledger({} as any)
      await ledger.connect()
      const res = await ledger.getExtendedPublicKey()
      expect(res.publicKey).toBe(zipPk)
    })

    it('if the return pk is compressed', async () => {
      getWalletExtendedPublicKeyMock.mockResolvedValueOnce({ public_key: zipPk, chain_code: '0x' })
      const ledger = new Ledger({} as any)
      await ledger.connect()
      const res = await ledger.getExtendedPublicKey()
      expect(res.publicKey).toBe(zipPk)
    })
  })
```

**Review by Keith-CY — APPROVED** [review]

File: packages/neuron-wallet/tests/services/hardware/ledger.test.ts (hunk 3 above)
Comment (Keith-CY) [reply packages/neuron-wallet/tests/services/hardware/ledger.test.ts:38]: Feel free to merge if these suggestions are not accepted @yanguoyu

**Pull request merged**